### ISSUE TITLE
NVDA mouse hover does not read aloud elements on Zoom Phone app

### ISSUE THREAD

**Issue by kschroeder-tamucc** [nvaccess/nvda#17683]
### Steps to reproduce:

Issues described here are occurring in other desktop and web applications, but we have a priority issue for employees with low vision using the Zoom Phone app and mouse hover functionality with the NVDA screen reader.

### Actual behavior:
Use mouse hover over Zoom Home in desktop application.
NVDA says "Unknown" or nothing at all on several buttons, including the top menu and the Home menu buttons.

Use mouse hover over Zoom Phone in desktop application. 
NVDA says "Unknown" or nothing at all for the phone buttons, though it can read the missed callers (History) when the mouse selects the caller element.

When receiving a phone call, NVDA will announce the caller but not if it is a transfer, etc. from the second line of text and does not read the buttons to accept or reject a phone call.

### Expected behavior:
Tested with Microsoft Narrator, and the elements mentioned above were read aloud upon mouse hover.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version: 
2024.4.2 (2024.4.2.35031)

#### Windows version: 
Microsoft Windows 11 Enterprise (10.0.22631 Build 22631)

#### Name and version of other software in use when reproducing the issue: 
Zoom Workplace Version: 6.3.6 (56144)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
N/A

#### If NVDA add-ons are disabled, is your problem still occurring?
N/A

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Inconsistent but leaning on still occurring across all elements. After restarting the issue came back in full.

**Comment by hwf1324** [comment]
Related: https://github.com/nvaccess/nvda/issues/17407#issuecomment-2544712156

CC: @codeofdusk, This could be a UIA related issue, using the method you provided, mouse tracking works properly.

@kschroeder-tamucc, I can create an `appModule` in the [Mouse Enhancement](https://github.com/hwf1324/mouseEnhancement) add-on if you want. Temporarily alleviate the problem using the method in the link above.

**Comment by kschroeder-tamucc** [comment]
I'd appreciate it or at least the chance to see if that works. Thanks!

**Comment by seanbudd** [comment]
Welcome @kschroeder-tamucc ,

This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by kschroeder-tamucc** [comment]
NV Access Software Developers,

We told them in a meeting last week on Thursday. Both sides tested and saw the same results during that meeting. However, Microsoft Narrator showed the content without any changes to the application. Information was properly exposed for almost all elements and we told them about that specific point as well.

That's why we brought this concern to your attention. This is one of several apps that is properly exposed but not read via mouse hover in NVDA. It is the primary issue, though, because it affects a major portion of an employee's job duties.

Thanks!

Karole Schroeder, CPACC
Digital Accessibility Officer
Office of Information Security
Texas A&M University - Corpus Christi
6300 Ocean Dr., Corpus Christi, TX 78412
P 361.825.3154 | E ***@***.******@***.***>
[Texas A&M University - Corpus Christi]

**Comment by hwf1324** [comment]
> I'd appreciate it or at least the chance to see if that works. Thanks!

@kschroeder-tamucc I've released v0.5.0 of the Mouse Enhancement Add-on, which should now be available for download via the Add-on Store.

**Comment by kschroeder-tamucc** [comment]
Thank you. I have asked my staff to review the add-on. In the meantime, Zoom provided an update on the one known element not exposed to assistive technology or keyboard tabbing within a phone call received: information on the "to" or "transfer" of the call (the text in the middle of a receiving phone call notification).

"this issue should be fixed in the next update of Zoom Version 6.4.0 released on 3/16."

**Comment by kschroeder-tamucc** [comment]
The client is using the workaround while waiting for the greater fix to the issue with NVDA. As mentioned above, Zoom plans to address the specific exposure to assistive technology issue in March.
